crates/hc_run_local_services/CHANGELOG.md

```diff
@@ -10,8 +10,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Improved documentation in README, code comments, help text, and error messages.
 - Upated from `structopt` 0.3 to `clap` 4.
 
-## 0.2.1-beta-dev.0
```

Comment:
we need to keep this heading

Reply:
Done, these were overwritten by the initial merge because these were never on develop